pkgs/desktops/gnome-3/core/gnome-tour/default.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv
+, rustPlatform
+, gettext
+, meson
+, ninja
+, fetchFromGitLab
+, pkgconfig
+, gtk3
+, glib
+, gdk-pixbuf
+, desktop-file-utils
+, appstream-glib
+, wrapGAppsHook
+, python3
+, gnome3
+, config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gnome-tour-unstable";
+  version = "2020-02-15";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = "gnome-tour";
+    rev = "129ec5d20a512ed4937da5088170c8ba61579c7b";
+    sha256 = "03if1y607jhy70rgb6bdicqj16c7khaa4wmiga7jrbbayf64nlgy";
+  };
+
+  cargoSha256 = "1gh2xvaq7v0zh375f98pbhjx65b07km47n2g5lplrxvznaw03c2a";
+
+  mesonFlags = [
+    "-Ddistro_name=NixOS"
+    "-Ddistro_icon_name=nix-snowflake"
+    "-Ddistro_version=20.03"
+  ];
+
+  nativeBuildInputs = [
+    pkgconfig
+    desktop-file-utils
+    meson
+    ninja
+    wrapGAppsHook
+    python3
+  ];
+
+  buildInputs = [
+    gtk3
+    glib
+    gdk-pixbuf
+    gettext
```

Comment:
Why?

Reply:
I looked at the gettext-rs crate and thought it wanted it. With some cursory research it appears it has its own one vendored.

Reply:
It would use the one from glibc on Linux if the feature is enabled:

https://github.com/Koka/gettext-rs/blob/a45ddb41d19e305b99211309bcac32f7fdbef1e8/gettext-sys/build.rs#L65-L82

It is not enabled by default:

https://github.com/Koka/gettext-rs/blob/a45ddb41d19e305b99211309bcac32f7fdbef1e8/gettext-sys/Cargo.toml#L13-L14
https://github.com/Koka/gettext-rs/blob/a45ddb41d19e305b99211309bcac32f7fdbef1e8/Cargo.toml#L22-L23

but fortunately gnome-tour does enable it:

https://gitlab.gnome.org/GNOME/gnome-tour/-/blob/5dc5ec9d47ab739ff1f2a5c1a732aa7e5483e400/Cargo.toml#L13